scripts/compatibility-test/readme.md

```diff
@@ -0,0 +1,38 @@
+﻿
+## Steps to configure & run compatibility test.
+
+Pre-requisites:
+   1. S3Server should be up and running 
+   
+Steps:
+
+1. Navigate to the compatibility-test script folder
+   ```
+      cd cortx-s3server/scripts/compatibility-test
+   ```
+
+2. create s3test.conf for the corresponding integration. Refer sample template for ceph and plunk integrations in the respective folders. Two s3 account credentials need to be updated in the configurations
+
+
+3. Execute run-test.sh with arguments 
+      
+      - Run ceph tests
+      ```
+         sh run-test.sh -c ceph/ceph.conf -i ceph
+      ```
+      - Run Splunk tests
+      
+      ```
+         sh run-test.sh -c splunk/splunk.conf -i splunk
+      ```
+      
+      - Run V2 or V4 test only 
+      ```
+         sh run-test.sh -c conf_location -i integration type (splunk | ceph) -t v4 
```

Comment:
Can be documented as 
"sh run-test.sh -c conf_location -i integration_type (i.e splunk | ceph) -t v4"

Reply:
Updated the sample cmd as per suggestion